libraries/lib-import-export/Import.cpp

```diff
@@ -629,6 +630,28 @@ bool Importer::Import(
    {
       // Try to open the file with this plugin (probe it)
       wxLogMessage(wxT("Opening with %s"),plugin->GetPluginStringID());
+
+#ifdef __WXMAC__
+      struct stat s;
+      memset(&s, 0, sizeof(struct stat));
+      auto err = stat(fName.data(), &s);
+      if(err != 0 || (S_ISREG(s.st_mode) && s.st_blocks == 0))
+      {
+         int dialogStyle = BasicUI::ProgressCanAbort | BasicUI::ProgressAppModal | BasicUI::ProgressShowElapsedTime | BasicUI::ProgressSmooth;
+         auto dialog = BasicUI::MakeGenericProgress({}, XO("Audacity"), XO("This may take several seconds"), dialogStyle);
```

Comment:
This should be `XO("Importing files"), XO("Importing %s...")` where %s is the file that currently is being imported